src/bicep-registry/scheduled-action/main.bicep

```diff
@@ -0,0 +1,138 @@
+// @resourceGroup @subscription @tenant
+
+// See https://learn.microsoft.com/rest/api/cost-management/scheduled-actions/create-or-update
+
+//===| Parameters |============================================================
+
+targetScope = 'subscription'
```

Comment:
The module supports all Bicep tooling and is optimized for subscription scope. Scope directives are specified throughout the doc to indicate differences between scopes.